optax/_src/transform.py

```diff
@@ -748,6 +748,44 @@ def update_fn(updates, state, params=None):
   return base.GradientTransformation(init_fn, update_fn)
 
 
+class ScaleByRpropState(NamedTuple):
+  nu: base.Updates
```

Comment:
Can we call it stepsizes or lrs to better understand what the variable is?

Reply:
Sure! This is done.